Sources/ApolloWebSocket/WebSocketTransport.swift

```diff
@@ -288,7 +289,7 @@ public class WebSocketTransport {
 
   public func closeConnection() {
     self.reconnect = false
-
+    self.reconnectWorkItem?.cancel()
```

Comment:
should it be set to nil after cancel()?

Reply:
Not mandatory but habit. I'll change that.